tasks/build.rake

```diff
@@ -10,7 +10,7 @@ namespace :vox do
 
     abort 'You must provide a platform.' if args[:platform].nil? || args[:platform].empty?
     platform = args[:platform]
-    os, _ver, arch = platform.match(/^(\w+)-([\w|\.]+)-(\w+)$/).captures
+    os, _ver, arch = platform.match(/^(\w+)-([\w|.]+)-(\w+)$/).captures
```

Comment:
Is it literal `.` or any char here? 🤔

Reply:
This `_ver` field looks strange in many more ways actually.. I just noticed it matches any of `\w` class or `|`/`.` literals it seems.. Not sure if matching `|` makes any sense in the platform version field..